provisioning_templates/provision/kickstart_default.erb

```diff
@@ -192,6 +192,15 @@ yum -t -y remove kernel-uek*
 sed -e 's/DEFAULTKERNEL=kernel-uek/DEFAULTKERNEL=kernel/g' -i /etc/sysconfig/kernel
 <% end -%>
 
+# This is RHEL7 specific code, as it uses systemctl, replace with
```

Comment:
Nit that's a bit late: The comment should be inside the if statement so you won't end up with just a comment when it's false. I'll submit a PR to fix this.